src/stories/Dropdown.stories.js

```diff
@@ -0,0 +1,48 @@
+import BasicDropdown from '../components/dropdown/BasicDropdown.vue'
+import TriggerDropdown from '../components/dropdown/TriggerDropdown.vue'
+
+import { action } from '@storybook/addon-actions'
+
+// This is required for each story
+export default { title: 'Dropdown' }
+
+// Customize components here.
+export const basicDropdown = () => ({
+  components: { BasicDropdown },
+  template: `
+  <div>
+    <el-dropdown>
+        <span class="el-dropdown-link">
+        More options<i class="el-icon-arrow-down el-icon--right"></i>
+        </span>
+        <el-dropdown-menu slot="dropdown">
+            <el-dropdown-item>Edit</el-dropdown-item>
+            <el-dropdown-item>Change data type</el-dropdown-item>
+            <el-dropdown-item disabled>For admins only</el-dropdown-item>
+            <el-dropdown-item divided>Delete</el-dropdown-item>
+        </el-dropdown-menu>
+    </el-dropdown>
```

Comment:
```
    <basic-dropdown>
      <template slot="items">
        <el-dropdown-item>Edit</el-dropdown-item>
        <el-dropdown-item>Change data type</el-dropdown-item>
        <el-dropdown-item disabled>For admins only</el-dropdown-item>
        <el-dropdown-item divided>Delete</el-dropdown-item>
      </template>
    </basic-dropdown>
```

Reply:
Thanks @michaelbukachi if we remove the `<span>` and hard code it in the component, does that then mean we need to change the label in the component each time? I.e. "More options" label? Or can we expose that on the stories.js file as well?

Reply:
We can use a prop to dynamically change the label.

Reply:
@michaelbukachi could you help suggest how I add a prop for the label?

Reply:
@michaelbukachi Actually, I was able to add an additional slot for the dropdown title. Let me know if this method is alright for exposing the label.